doc/modules/linear_model.rst

```diff
@@ -903,15 +903,21 @@ in these settings.
 
 .. topic:: **Trade-offs: which estimator?**
 
-   Scikit-learn provides 2 robust regression estimators:
+   Scikit-learn provides 3 robust regression estimators:
    :ref:`RANSAC <ransac_regression>` and
    :ref:`Theil Sen <theil_sen_regression>`
+   :ref:`HuberRegressor <huber_regression>`
 
-   * :ref:`RANSAC <ransac_regression>` is faster, and scales much better
-     with the number of samples
+  * :ref:`HuberRegressor` <huber_regression> should be faster than
```

Comment:
I wish it would be so easy to predict heuristics for what `n_samples` I should use what estimator. This depends on HUGE number of factors. 

RANSAC would definitely be faster for `n_samples` >> `n_features` since it produces 100 fits but it is likely to be less robust because of the smaller number of samples doing each fit. The same argument goes to TheilSenRegressor because the smaller your `n_subsamples` is, it is going to be less robust.

We could just say that TheilSenRegressor and RANSAC are to be faster but they are unlikely to be as robust as HuberRegressor for the default parameters when n_samples >> n_features. How does that sound?